server/data/data-api/src/test/java/org/apache/james/sieverepository/api/SieveRepositoryManagementMBeanTest.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.james.sieverepository.api;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class SieveRepositoryManagementMBeanTest {
+
+    @Test
+    public void testSanitizeFileNameWithValidInputForMBean() {
```

Comment:
these tests are duplication of other ones but you didn't changed them to reflect changes in the other test suite

Reply:
Right. My bad.

Reply:
Done.